lib/ansible/modules/cloud/amazon/aws_application_scaling_policy.py

```diff
@@ -170,7 +178,56 @@ def delete_scaling_policy(connection, module):
         except Exception as e:
             module.fail_json(msg=str(e), exception=traceback.format_exc())
 
-    module.exit_json(changed=changed)
+    result = {"changed": changed}
+    return result
+
+
+def create_scalable_target(connection, module):
+    changed = False
+
+    scalable_targets = connection.describe_scalable_targets(
+        ServiceNamespace=module.params.get('service_namespace'),
+        ResourceIds=[
+            module.params.get('resource_id'),
+        ],
+        ScalableDimension=module.params.get('scalable_dimension')
+    )
+
+    if (
+        not scalable_targets['ScalableTargets']
+        or scalable_targets['ScalableTargets'][0]['MinCapacity'] != module.params.get('minimum_tasks')
+        or scalable_targets['ScalableTargets'][0]['MaxCapacity'] != module.params.get('maximum_tasks')
+    ):
+        changed = True
+        try:
+            connection.register_scalable_target(
+                ServiceNamespace=module.params.get('service_namespace'),
+                ResourceId=module.params.get('resource_id'),
+                ScalableDimension=module.params.get('scalable_dimension'),
+                MinCapacity=module.params.get('minimum_tasks'),
+                MaxCapacity=module.params.get('maximum_tasks')
+            )
+        except Exception as e:
```

Comment:
It looks like the rest of the module is using `except Exception` so you've probably done this for consistency, but it would be better to catch only the specific exceptions, which would be the botocore.exceptions ClientError and BotoCoreError. Same for the one below. At some point the other 'except Exception's need to be removed.
Guidelines on exception handling: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#using-fail_json-and-avoiding-ansiblemodule_utilsawscore